ddcz/templates/creative-pages/item-list.html

```diff
@@ -20,43 +20,43 @@ <h1 class="page-heading">{{ heading }}</h1>
+            <h2><a href="{% url 'ddcz:creation-detail' creative_page_slug=creative_page_slug creation_id=a.id creation_slug=a.get_slug %}">{{ a.name }}</a>{% creation_rating a.rating skin %}</h2>
             <div class="table-like">
-                <p><span class="label">Skupina:</span> <span class="value">{{ a.skupina }}</span></p>
-                {% if a.uc %}
-                    <p><span class="label">Síla zbraně:</span> <span class="value">{{ a.uc }}</span></p>
+                <p><span class="label">Skupina:</span> <span class="value">{{ a.group }}</span></p>
+                {% if a.attack_number %}
+                    <p><span class="label">Síla zbraně:</span> <span class="value">{{ a.attack_number }}</span></p>
                 {% endif %}
-                {% if a.kz %}
-                    <p><span class="label">Kvalita zbroje:</span> <span class="value">{{ a.kz }}</span></p>
+                {% if a.armour_quality %}
+                    <p><span class="label">Kvalita zbroje:</span> <span class="value">{{ a.armour_quality }}</span></p>
                 {% endif %}
-                {% if a.delka %}
-                    <p><span class="label">Délka:</span> <span class="value">{{ a.delka }}</span></p>
+                {% if a.length %}
+                    <p><span class="label">Délka:</span> <span class="value">{{ a.length }}</span></p>
                 {% endif %}
-                {% if a.cena %}
-                    <p><span class="label">Cena:</span> <span class="value">{{ a.cena }}</span></p>
+                {% if a.price %}
+                    <p><span class="label">Cena:</span> <span class="value">{{ a.price }}</span></p>
                 {% endif %}
-                {% if a.malydostrel %}
-                    <p><span class="label">Malý dostřel:</span> <span class="value">{{ a.malydostrel }}</span></p>
+                {% if a.range_small %}
+                    <p><span class="label">Malý dostřel:</span> <span class="value">{{ a.range_small }}</span></p>
                 {% endif %}
-                {% if a.strednidostrel %}
-                    <p><span class="label">Střední dostřel:</span> <span class="value">{{ a.strednidostrel }}</span></p>
+                {% if a.range_expected %}
+                    <p><span class="label">Střední dostřel:</span> <span class="value">{{ a.range_expected }}</span></p>
                 {% endif %}
-                {% if a.velkydostrel %}
-                    <p><span class="label">Velký dostřel:</span> <span class="value">{{ a.velkydostrel }}</span></p>
+                {% if a.range_long %}
+                    <p><span class="label">Velký dostřel:</span> <span class="value">{{ a.range_long }}</span></p>
                 {% endif %}
-                {% if a.sfera %}
-                    <p><span class="label">Sféra:</span> <span class="value">{{ a.sfera }}</span></p>
+                {% if a.sphere %}
```

Comment:
Just thinking of the possibility to call it a plane. They use Spiritual Sphere in English? I remember hearing more of Spiritual or Demon's Plane.

Reply:
Oh that's a good point, lemme fix